apps/app/dashboards/government-programs/peoples-income-initiative/index.tsx

```diff
@@ -1,37 +1,257 @@
-import AgencyBadge from "@components/Badge/agency";
-import { Hero } from "@components/index";
-import { useTranslation } from "@hooks/useTranslation";
-import { FunctionComponent } from "react";
-import Container from "@components/Container";
+import { FunctionComponent, useRef } from "react";
+import dynamic from "next/dynamic";
+import {
+  AgencyBadge,
+  Container,
+  Dropdown,
+  Hero,
+  LeftRightCard,
+  Section,
+  StateDropdown,
+} from "@components/index";
 import { IPREPUIcon } from "@components/Icon/agency";
+import Slider from "@components/Chart/Slider";
+import { SliderProvider } from "@components/Chart/Slider/context";
+import { OptionType } from "@components/types";
+import ArrowRightIcon from "@heroicons/react/20/solid/ArrowRightIcon";
+import { useData } from "@hooks/useData";
+import { useSlice } from "@hooks/useSlice";
+import { useTranslation } from "@hooks/useTranslation";
+import { AKSARA_COLOR, CountryAndStates } from "@lib/constants";
+import { getTopIndices, numFormat, toDate } from "@lib/helpers";
+import { routes } from "@lib/routes";
 
 /**
  * PeoplesIncomeInitiative Dashboard
  * @overview Status: In-development
  */
 
-interface PeoplesIncomeInitiativeProps {}
+const Choropleth = dynamic(() => import("@components/Chart/Choropleth"), { ssr: false });
+const Timeseries = dynamic(() => import("@components/Chart/Timeseries"), { ssr: false });
+
+interface PeoplesIncomeInitiativeProps {
+  choropleth: any;
+  last_updated: any;
+  params: { state: string };
+  timeseries: any;
+  timeseries_callout: any;
+}
 
-const PeoplesIncomeInitiative: FunctionComponent<PeoplesIncomeInitiativeProps> = ({}) => {
+const PeoplesIncomeInitiative: FunctionComponent<PeoplesIncomeInitiativeProps> = ({
+  choropleth,
+  last_updated,
+  params,
+  timeseries,
+  timeseries_callout,
+}) => {
   const { t, i18n } = useTranslation(["dashboard-peoples-income-initiative", "common"]);
+  const currentState = params.state;
+  const FILTER_OPTIONS: Array<OptionType> = [
+    "absolute",
+    "absolute_hardcore_poor",
+    "per_capita",
+    "per_capita_hardcore_poor",
+  ].map((key: string) => ({
+    label: t(key),
+    value: key,
+  }));
+  const { data, setData } = useData({
+    minmax: [0, timeseries.data.x.length],
+    filter: FILTER_OPTIONS[0],
+  });
+  const { coordinate } = useSlice(timeseries.data, data.minmax);
+  const INITIATIVE = ["intan", "insan", "ikhsan"];
+  const topStateIndices = getTopIndices(choropleth.data[data.filter.value].y.value, 3, true);
```

Comment:
this somehow includes Malaysia in the ranking, when it should just be the states. Check again